phpini/phpini-lib.pl

```diff
@@ -919,14 +928,14 @@ sub list_php_base_packages
 sub list_all_php_module_packages
 {
 my ($base) = @_;
-$base =~ s/(-php|-runtime)$//;
+$base =~ s/-common$//;
```

Comment:
What if the base package name contains -php ?

Reply:
> What if the base package name contains -php ?

Considering we are passing a package name only with `-common` like `php8.4-common` or `php-common` or `php83-php-common` by removing the last `-common` are are getting exactly the package name we want.